src/rdo.rs

```diff
@@ -797,7 +797,7 @@ pub fn rdo_mode_decision<T: Pixel>(
   fi: &FrameInvariants<T>, ts: &mut TileStateMut<'_, T>,
   cw: &mut ContextWriter, bsize: BlockSize, tile_bo: TileBlockOffset,
   pmvs: &mut [Option<MotionVector>], inter_cfg: &InterConfig,
-) -> RDOPartitionOutput {
+) -> PartitionParameters {
```

Comment:
Input to this function is a 'block', i.e, one partition, hence I expect the reader assumes that the output from this function is rdo results of one 'partition'. So, I doubt we need include 'partition' as part of the output name?

Reply:
What is the problem? The struct contains the parameters for coding a single partition, so if it is not part of the name there is missing information. Specifying it makes the code self-documenting, and if it were called "Parameters" and used elsewhere in the program, it would be impossible to tell what kind of data is inside at a glance.